packages/datetime/test/dateRangePickerTests.tsx

```diff
@@ -16,7 +16,7 @@ import * as Errors from "../src/common/errors";
 import { Months } from "../src/common/months";
 import { Classes as DateClasses, DateRange, DateRangePicker, IDateRangePickerProps } from "../src/index";
 
-describe("<DateRangePicker>", () => {
+xdescribe("<DateRangePicker>", () => {
```

Comment:
Reasoning behind `xdescribe` vs. `describe.skip`? Reads more clearly if it'll behave equivalently for you.